src/Tomcat/TomcatController.ts

```diff
@@ -57,6 +58,17 @@ export class TomcatController {
         vscode.window.showTextDocument(vscode.Uri.file(configFile), { preview: false });
     }
 
+    public async browseWarPackage(warPackage: WarPackage): Promise<void> {
+        if (warPackage) {
+            const server: TomcatServer = this._tomcatModel.getTomcatServer(warPackage.serverName);
+            opn(await this.getServerUri(server, warPackage.label));
+            if (!server.isStarted()) {
+                server.outputChannel.appendLine('Start server to see real war package');
```

Comment:
Why not prompt the user to start the server using actions?

Reply:
good idea